LEAF_Nexus/api/controllers/TagController.php

```diff
@@ -39,7 +39,7 @@ public function post($act)
     	$this->index['POST'] = new ControllerMap();
     
     	$this->index['POST']->register('tag/[text]/parent', function($args) use ($tag) {
-    		return $tag->setParent($args[0], $_POST['parentTag']);
+    		return $tag->setParent($args[0], $tag->sanitizeInput($_POST['parentTag']));
```

Comment:
This should use the sanitize functions in `XSSHelpers.php`. `Tag.php` uses the sanitize function in `Data.php`, which is an old version of the sanitation functions.